interface/forms/care_plan/new.php

```diff
@@ -148,99 +166,111 @@ function set_related(codetype, code, selector, codedesc) {
             });
 
         </script>
-        <p><span class="forms-title"><?php echo xlt('Care Plan Form'); ?></span></p>
-        </br>
-        <?php echo "<form method='post' name='my_form' " . "action='$rootdir/forms/care_plan/save.php?id=" . attr($formid) . "'>\n"; ?>
-        <table id="care_plan" border="0">
-
-            <?php
-            if (!empty($check_res)) {
-                foreach ($check_res as $key => $obj) {
+        <div class="container">
+
+            <div class="row">
+                <div class="page-header">
+                        <h2><?php echo xlt('Care Plan Form'); ?></h2>
+                </div>
+            </div>
+            <div class="row">
+            <?php echo "<form method='post' name='my_form' " . "action='$rootdir/forms/care_plan/save.php?id=" . attr($formid) . "'>\n"; ?>
```

Comment:
Would be better to break out the `<form>` to regular HTML and just inject the PHP echo where needed, even it that means multiple time on the line. Or just create a separate line for the action path and do 1 echo